packages/now-build-utils/src/detectors/next.ts

```diff
@@ -0,0 +1,22 @@
+import { DetectorParameters, DetectorResult } from '../types';
+
+export default async function detectNext({
+  fs: { hasDependency },
+}: DetectorParameters): Promise<DetectorResult> {
+  const hasNext = await hasDependency('next');
+  if (!hasNext) return false;
+  return {
+    buildCommand: ['next', 'build'],
+    buildDirectory: 'build',
+    devCommand: ['next', 'start', '-p', '$PORT'],
+    routes: [
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        dest: '/index.html',
```

Comment:
I'm not sure about this one. I'll defer to @Timer and @dav-is

Reply:
I'll remove. Was probably copy+paste leftover. I'm not even sure that Next.js should be a detector considering we have a dedicated `@now/next` builder 🤔